packages/atomic/src/components/common/switch.tsx

```diff
@@ -13,7 +13,7 @@ export const Switch: FunctionalComponent<SwitchProps> = (props) => {
   const attributes = {
     onClick: () => props.onToggle?.(!props.checked),
     'arial-label': props.ariaLabel,
-    'aria-checked': props.checked,
+    'aria-checked': props.checked ? 'true' : 'false',
```

Comment:
Perhaps a falsey check would achieve the same thing (didn't test)?

```suggestion
    'aria-checked': !!props.checked,
```

Reply:
```suggestion
    'aria-checked': String(props.checked),
```

Reply:
I believe the point is it has to be a string

Reply:
```suggestion
    'aria-checked': String(!!props.checked),
```
This then 😛
jk